tests/unit/test_cli.py

```diff
@@ -142,7 +142,7 @@ def side_effect(whatever):
                 False,
                 "config.yaml",
                 "",
-                "Usage: cachi2 [OPTIONS] COMMAND [ARGS]...\nTry 'cachi2 --help' for help.\n\nError: Invalid value for '--config-file': File 'config.yaml' does not exist.\n",
+                "Usage: cachi2 [OPTIONS] COMMAND [ARGS]...\nTry 'cachi2 --help' for help.\n",
```

Comment:
We're not checking the specific error message anymore, only the generic one. We should keep checking the specific error.

(same for most of the other changes in this file)